automation/taskcluster/decision_task.py

```diff
@@ -131,6 +131,45 @@ def release(channel, is_staging, version_name):
         signing_task_id,
         apks=apk_paths,
         channel=channel,
+        # TODO until org.mozilla.fenix.nightly is made public, put it on the internally-testable track
+        override_google_play_track=None if channel != "nightly" else "internal",
+        is_staging=is_staging,
+    )
+
+    return (build_tasks, signing_tasks, push_tasks)
+
+
+def nightly_to_production_app(is_staging, version_name):
```

Comment:
This function is very very similar to the one that [used to be here](https://github.com/mozilla-mobile/fenix/pull/3740/files#diff-168d551d2c0830884a5caeb50ae576e9L140)